examples/kubecon-na-22-demo/README.md

```diff
@@ -0,0 +1,39 @@
+# KubeCon 2022 NA Demo
+
+This demonstration is based on the [Observability Example](../observability) and should do the following:
+
+![img.png](assets/big-picture.png)
+
+## Prepare Secret for Slack Notification
+> kubectl create secret generic slack-notification --from-literal=SECURE_DATA='{"slack_hook":"<hook>","text":"Deployed PodTatoHead Application"}' -n podtato-kubectl -oyaml --dry-run > base/slack-secret.yaml
```

Comment:
In the preview the `<hook>` part is empy, I guess it is due to the renderer trying to parse it as magic URL
```suggestion
> kubectl create secret generic slack-notification --from-literal=SECURE_DATA='{"slack_hook":"\<hook>","text":"Deployed PodTatoHead Application"}' -n podtato-kubectl -oyaml --dry-run > base/slack-secret.yaml
```